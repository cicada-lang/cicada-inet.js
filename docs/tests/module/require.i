require "../datatypes/Nat.i"

begin zero one add inspect run inspect end
