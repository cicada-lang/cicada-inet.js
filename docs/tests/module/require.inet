require "../datatypes/Nat.i"

begin zero one add run inspect end