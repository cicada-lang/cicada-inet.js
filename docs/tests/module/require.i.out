(add1₁₇)-value!target-(add₆)
(zero₁₅)-value addend-(add₆)
(zero₁₆)-value prev-(add1₁₇)
(add₆)-return

(zero₁₅)-value prev-(add1₁₈)
(add1₁₈)-value

